src/read/dwarf.rs

```diff
@@ -671,6 +684,11 @@ impl<R: Reader> Unit<R> {
                             unit.rnglists_base = base;
                         }
                     }
+                    constants::DW_AT_GNU_dwo_id => {
+                        if let AttributeValue::DwoId(dwo_id) = attr.value() {
+                            unit.dwo_id = Some(dwo_id);
```

Comment:
Should we check to see if this is already present? DW_AT_GNU_dwo_id shouldn't really be present in DWARF 5 programs.

Reply:
Ok. It shouldn't matter, but it doesn't take much to add the check.